SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -98,6 +98,7 @@ sed -i 's@DICTPATH.*@DICTPATH\t/usr/share/cracklib/pw_dict@' \
 install -vdm 755 %{buildroot}/bin
 install -vdm755 %{buildroot}%{_sysconfdir}/default
 mv -v %{buildroot}%{_bindir}/passwd %{buildroot}/bin
+chmod ug-s %{buildroot}/bin/passwd
```

Comment:
I think you could combine these two lines into a single call to `install`, might be worth looking into.